### DISCUSSION TITLE
Difference between VSCode themes

### DISCUSSION THREAD

**Discussion by jvzaniolo** [daltonmenezes/aura-theme#30] [Q&A] [answered]
Hey, @daltonmenezes nice work!

Could you explain in the README for VSCode, the difference between Aura Dark Plus, Aura Dark, and Aura Soft or is it just the background color?

**Comment by daltonmenezes** [comment, marked as answer]
Hi @jvzaniolo , you right, the main difference around `Aura Dark Plus`, `Aura Dark`, and `Aura Soft` is the UI colors and soft text variants are about color contrasts. 

I'm working now on Aura v2.0 and `Aura Dark Plus` will  be the **default theme** and renamed as `Aura Dark` replacing the current one. The current `Aura Dark` will be removed. That been said, this will be the the new list:
  - Aura Dark
  - Aura Dark (Soft Text)
  - Aura Soft Dark
  - Aura Soft Dark (Soft Text)
